parser.py

```diff
@@ -1,240 +1,18 @@
-from collections import namedtuple
+from enum import Enum
 import sympy
 
 from lark import Lark, Transformer
 
+from parser_utils import *
```

Comment:
Moved all these here

Reply:
Will need to remove the `*` import when we add linting 😆